test/spmd/tensor/test_dtensor.py

```diff
@@ -192,6 +199,40 @@ def test_dtensor_spec_read_only_after_set(self):
         self.assertTrue(sharded_tensor.placements is not shard_spec)
         self.assertNotEqual(sharded_tensor.placements, shard_spec)
 
+    @with_comms
+    def test_dtensor_spec_local_shard_offset(self):
+        device_mesh = DeviceMesh(
```

Comment:
It should be possible to write a test that uses local_offset and local_shape to construct a ShardedTensor out of a DT that is only sharded, right?

Reply:
should be, local_offset + local_shape should be sufficient to construct ShardedTensor's ShardMetadata except the placement device.